installing/installing_alibaba/installing-alibaba-assisted-installer.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-alibaba-assisted-installer"]
+= Installing a cluster on Alibaba Cloud using the Assisted Installer
+include::_attributes/common-attributes.adoc[]
+:context: installing-alibaba-assisted-installer
+
+toc::[]
+
+From {product-title} 4.16 and later, you can use the {ai-full} to install an {product-title} cluster on {alibaba}. {alibaba} provides a broad range of cloud computing and data storage services to online businesses and global enterprises.
+
+:FeatureName: Installing {alibaba} with {ai-full}
+include::snippets/technology-preview.adoc[]
+
+include::modules/alibaba-ai-installing.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For the {ai-full} procedures, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/installing_openshift_container_platform_with_the_assisted_installer/index[Installing {product-title} with the {ai-full}].
+
+include::modules/alibaba-ai-converting-image-to-qcow2.adoc[leveloffset=+1]
+
+.Additional resources
+
+*  To check if your CPU supports the required virtualization flags, see link:https://www.cyberciti.biz/faq/linux-xen-vmware-kvm-intel-vt-amd-v-support/[Find Out if CPU Supports Intel VT/AMD Virtualization for KVM in Linux]
```

Comment:
Is this a trustworthy/approved site? Check with Product Experience if you aren't 1000% sure about it.